cmd/cluster/sre_operators/describeCmd.go

```diff
@@ -0,0 +1,331 @@
+  clients to fetch the latest versions of each operator from its respective repository.
+	`
+)
+
+func newCmdDescribe(streams genericclioptions.IOStreams, client client.Client) *cobra.Command {
+	opts := &sreOperatorsDescribeOptions{
+		kubeCli:   client,
+		IOStreams: streams,
+	}
+
+	describeCmd := &cobra.Command{
+		Use:     "describe",
+		Short:   "Describe SRE operators",
+		Long:    sreOperatorsLongDescription,
+		Example: sreOperatorsDescribeExample,
+		Args:    cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			util.CheckErr(opts.checks(cmd))
+			output, _ := opts.DescribeOperator(cmd, args[0])
+			util.CheckErr(opts.printText(output))
+		},
+	}
+
+	return describeCmd
+}
+
+func (ctx *sreOperatorsDescribeOptions) checks(cmd *cobra.Command) error {
+	if _, err := config.GetConfig(); err != nil {
+		return util.UsageErrorf(cmd, "could not find KUBECONFIG, please make sure you are logged into a cluster")
+	}
+
+	return nil
+}
+
+func (ctx *sreOperatorsDescribeOptions) printText(output []sreOperatorDetails) error {
+	p := printer.NewTablePrinter(ctx.IOStreams.Out, 20, 1, 3, ' ')
+
+	for _, op := range output {
+		p.AddRow([]string{"Operator Name:", op.Name})
+		if op.Current != op.Expected {
+			p.AddRow([]string{"Current Version:", Red + op.Current + " (outdated)" + RestoreColor})
+			p.AddRow([]string{"Expected Version:", op.Expected})
+		} else {
+			p.AddRow([]string{"Current Version:", op.Current})
+			p.AddRow([]string{"Expected Version:", op.Expected})
+		}
+		p.AddRow([]string{"Channel:", op.Channel})
+		p.AddRow([]string{"CSV Status:", op.CsvStatus})
+		p.AddRow([]string{"CSV Phase:", op.CsvHealthPhase})
+		p.AddRow([]string{"Latest CSV Health Message:", op.CsvHealthMessage})
+		p.AddRow([]string{"Latest Subscription Message:", op.SubscriptionMessage})
+		p.AddRow([]string{"OperatorGroup Status:", op.OperartorGroupHealth})
+		p.AddRow([]string{"Deployment Status:", op.DeploymentHealth})
+		p.AddRow([]string{"Pod Status:", op.PodHealth})
+	}
+
+	if err := p.Flush(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (ctx *sreOperatorsDescribeOptions) DescribeOperator(cmd *cobra.Command, operatorName string) ([]sreOperatorDetails, error) {
+	listOfOperators := []string{
+		"openshift-addon-operator",
+		"aws-vpce-operator",
+		"openshift-custom-domains-operator",
+		"openshift-managed-node-metadata-operator",
+		"openshift-managed-upgrade-operator",
+		"openshift-must-gather-operator",
+		"openshift-ocm-agent-operator",
+		"openshift-osd-metrics",
+		"openshift-rbac-permissions",
+		"openshift-splunk-forwarder-operator",
+		"aws-account-operator",
+		"certman-operator",
+		"openshift-cloud-ingress-operator",
+		"openshift-monitoring",
+		"deadmanssnitch-operator",
+		"openshift-deployment-validation-operator",
+		"gcp-project-operator",
+		"openshift-velero",
+		"pagerduty-operator",
+		"openshift-route-monitor-operator",
+		"openshift-observability-operator",
+	}
+
+	listOfOperatorNames := []string{
```

Comment:
Is it possible if we can get this list dynamically from repositories or somewhere?